packages/e2e-tests/src/specs/editor/fontCheck/fontCheckMetrics.js

```diff
@@ -118,11 +118,15 @@ describe('Font Check Metrics', () => {
     mockResponse = undefined;
   });
 
-  afterAll(() => {
+  afterAll(async () => {
+    await page.setRequestInterception(false);
     stopRequestInterception();
   });
 
-  it('should receive updated font metrics and not alter history', async () => {
+  jest.retryTimes(3);
```

Comment:
Maybe move this to the top of the describe block or to the top of the file?

Also, should we try `jest.retryTimes(3, {logErrorsBeforeRetry: true});`?